content/docs/command-reference/push.md

```diff
@@ -18,55 +18,44 @@ positional arguments:
 ## Description
 
 The `dvc pull` and `dvc push` commands are the means for uploading and
-downloading data to and from remote storage. These commands are similar to
-`git pull` and `git push`, respectively (with some key differences given the
-nature of DVC, see details below).
+downloading data to and from remote storage (S3, SSH, GCS, etc.). These commands
+are similar to `git pull` and `git push`, respectively.
 
 [Data sharing](/doc/use-cases/sharing-data-and-model-files) across environments,
 and preserving data versions (input datasets, intermediate results, models,
-[metrics](/doc/command-reference/metrics), etc.)
-[remotely](/doc/command-reference/remote) are the two most common use cases for
-these commands.
+[metrics](/doc/command-reference/metrics), etc.) remotely are the most common
+use cases for these commands.
 
-The `dvc push` command allows us to upload data to remote storage. It doesn't
-save any changes to the code, `dvc.yaml`, or `.dvc` files (those should be saved
-with `git commit` and `git push`).
+The `dvc push` command uploads data to
+[remote storage](/doc/command-reference/remote). It doesn't save any changes to
+the code, `dvc.yaml`, or `.dvc` files (that should be saved with `git commit`
```

Comment:
should include `dvc.lock` here ...

Reply:
True. This whole sentence is kind of a note though, now that I think about it. I changed it a little in 4717b78b. It's still not perfect but we'll have to review this again soon, when addressing #1663.